src/sonic-yang-models/tests/yang_model_tests/tests/loopback.json

```diff
@@ -1,7 +1,7 @@
 {
-    "LOOPBACK_IPPREFIX_PORT_MUST_CONDITION_FALSE": {
-	"desc": "Loopback Ip-prefix port-name must condition failure.",
-	"eStrKey" : "Must"
+    "LOOPBACK_IPPREFIX_PORT_FOR_NON_EXIST_INTERFACE": {
+	"desc": "Configure Loopback Ip-prefix for non-existing Loopback interface.",
```

Comment:
>	 [](http://example.com/codeflow?start=0&length=1)

Replace tab with 4 spaces. Please format the whole file's indentation.

Reply:
updated